packages/url_launcher/url_launcher/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.8.0-nullsafety
```

Comment:
why not `6.0.0`?

Reply:
I updated the version number in the middle of the discussion we had for the `url_launcher_platform_interface`. Since for that plugin we settled on doing a major release I think it should be good to do it here as well. 

So I updated the version to 6.0.0-nullsafety. Main argument the `canLaunch` and `launch` methods won't be returning an optional `null` value anymore (where in earlier versions this was a possibility).